.github/workflows/ci.yml

```diff
@@ -124,12 +124,15 @@ jobs:
 
   import-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-env: [3.8, 3.9, '3.10', '3.11']
```

Comment:
Some have `''`, other don't, does it make a difference?

Reply:
fixed